src/Ledger/Utxo.lagda

```diff
@@ -45,19 +37,6 @@ instance
 
   HasCoin-Map : ∀ {A} → ⦃ DecEq A ⦄ → HasCoin (A ⇀ Coin)
   HasCoin-Map .getCoin s = Σᵐᵛ[ x ← s ᶠᵐ ] x
-
-
--- utxoEntrySizeWithoutVal = 27 words (8 bytes)
-utxoEntrySizeWithoutVal : MemoryEstimate
-utxoEntrySizeWithoutVal = 8
-
-utxoEntrySize : TxOut → MemoryEstimate
-utxoEntrySize (fst , v) = utxoEntrySizeWithoutVal + size v
-
--- TODO: fix this
-serSize : Value → MemoryEstimate
-serSize = λ _ → zero
-
```

Comment:
`utxoEntrySizeWithoutVal`, `utxoEntrySize`, `serSize` are not used here.  Were they inserted to support some planned functionality?  I'm a bit hesitant to remove them... yet `Everything` still type-checks without them, so... should we leave them there but comment them out for now?

Reply:
Yeah, this is from the Shelley-MA spec. They'll get used eventually, so I think it's best to just leave them in.

Reply:
I restored the three definitions mentioned above.  They are exactly as before (in particular, uncommented).